<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inExperimentUi</class>
 <widget class="QWidget" name="inExperimentUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1600</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>1080</width>
    <height>1600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblWaittingGif">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>570</width>
     <height>570</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPauseExperi">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>1320</y>
     <width>97</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblNoticeText_1st">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>930</y>
     <width>500</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNoticeText_2nd">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>1060</y>
     <width>500</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNoticeText_3rd">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>990</y>
     <width>500</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
